com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/HTTPParallelRequest.spl

```diff
@@ -0,0 +1,134 @@
+namespace com.ibm.streamsx.inet.http;
+
+/**
+ * The operator HTTPParallelRequest implements parallel http operations and processes the results asynchronous from the input stream.
```

Comment:
Suggest as the summary:

Issues HTTP requests in parallel asynchronously from the input stream.

Reply:
ok